### DISCUSSION TITLE
更新1.35版本后，导入plots后但是不耕种，check显示plots 有效

### DISCUSSION THREAD

**Discussion by 781747217** [Chia-Network/chia-blockchain#11915] [Support]
What happened?
更新1.35版本后，通过chia  plots add -d 命令导入plots图的地址，并通过chia plots check检查存在有效plots，但是通过chia farm summary命令却显示no plots，请问是我的操作有问题吗？

Version
1.3.5

What platform are you using?
Ubuntu 20.04

What ui mode are you using?
CLI

Relevant log output
No response

**Comment by loppefaaret** [comment]
Could you try and see if there are any WARNING or ERROR reported by the harvester in your `debug.log` found in your users home folder, in `~/.chia/mainnet/log/`
